checks/evaluation/ci_tests.go

```diff
@@ -16,122 +16,62 @@ package evaluation
 
 import (
 	"fmt"
-	"strings"
 
 	"github.com/ossf/scorecard/v4/checker"
+	sce "github.com/ossf/scorecard/v4/errors"
 	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/testsRunInCI"
 )
 
-const (
-	// CheckCITests is the registered name for CITests.
-	CheckCITests = "CI-Tests"
-	success      = "success"
-)
-
-func CITests(_ string, c *checker.CITestData, dl checker.DetailLogger) checker.CheckResult {
-	totalMerged := 0
-	totalTested := 0
-	for i := range c.CIInfo {
-		r := c.CIInfo[i]
-		totalMerged++
-
-		var foundCI bool
-
-		// GitHub Statuses.
-		prSuccessStatus, err := prHasSuccessStatus(r, dl)
-		if err != nil {
-			return checker.CreateRuntimeErrorResult(CheckCITests, err)
-		}
-		if prSuccessStatus {
-			totalTested++
-			foundCI = true
-			continue
-		}
-
-		// GitHub Check Runs.
-		prCheckSuccessful, err := prHasSuccessfulCheck(r, dl)
-		if err != nil {
-			return checker.CreateRuntimeErrorResult(CheckCITests, err)
-		}
-		if prCheckSuccessful {
-			totalTested++
-			foundCI = true
-		}
+const CheckCITests = "CI-Tests"
 
-		if !foundCI {
-			// Log message says commit, but really we only care about PRs, and
-			// use only one commit (branch HEAD) to refer to all commits in a PR
+func CITests(name string,
+	findings []finding.Finding,
+	dl checker.DetailLogger,
+) checker.CheckResult {
+	expectedProbes := []string{
+		testsRunInCI.Probe,
+	}
+	if !finding.UniqueProbesEqual(findings, expectedProbes) {
+		e := sce.WithMessage(sce.ErrScorecardInternal, "invalid probe results")
+		return checker.CreateRuntimeErrorResult(name, e)
+	}
 
+	// Debug PRs that were merged without CI tests
+	for i := range findings {
+		f := &findings[i]
+		if f.Outcome == finding.OutcomeNegative {
 			dl.Debug(&checker.LogMessage{
-				Text: fmt.Sprintf("merged PR %d without CI test at HEAD: %s", r.PullRequestNumber, r.HeadSHA),
+				Text: f.Message,
 			})
 		}
 	}
 
-	if totalMerged == 0 {
+	// check that the project has pull requests
+	if noPullRequestsFound(findings) {
 		return checker.CreateInconclusiveResult(CheckCITests, "no pull request found")
 	}
 
-	reason := fmt.Sprintf("%d out of %d merged PRs checked by a CI test", totalTested, totalMerged)
-	return checker.CreateProportionalScoreResult(CheckCITests, reason, totalTested, totalMerged)
-}
+	totalMerged := findings[0].Values["totalMerged"]
+	totalTested := findings[0].Values["totalTested"]
```

Comment:
similar comment as webhooks: now that we have 1 finding per PR, we don't need these values